ui-speedspacechart/src/components/utils.ts

```diff
@@ -9,6 +9,7 @@ type SpeedRangeValues = {
 type MaxPositionValues = {
   maxPosition: number;
   RoundMaxPosition: number;
+  intermediateTicksPosition: number;
 };
 
 /**
```

Comment:
It would be nice to add some tests for these utils functions

Reply:
done